scripts/release/update_jira.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+
+import json, time, logging, getpass
+from optparse import OptionParser
+from utils import jira_helper
```

Comment:
Good point. Just checked the style guide and it's to have imports in separate lines.